src/main/java/com/piekoszek/nowaksharedrent/invite/InviteController.java

```diff
@@ -0,0 +1,45 @@
+package com.piekoszek.nowaksharedrent.invite;
+
+import com.piekoszek.nowaksharedrent.jwt.JwtData;
+import com.piekoszek.nowaksharedrent.jwt.JwtService;
+import com.piekoszek.nowaksharedrent.response.MessageResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+class InviteController {
+
+    private InviteService inviteService;
+    private JwtService jwtService;
+
+    InviteController(InviteService inviteService, JwtService jwtService) {
+        this.inviteService = inviteService;
+        this.jwtService = jwtService;
+    }
+
+    @PostMapping("/invite")
+    ResponseEntity<Object> createInvitation(@RequestBody Invitation invitation, @RequestHeader("Authorization") String token) {
+        JwtData jwtData = jwtService.readToken(token);
+        inviteService.createInvitation(jwtData.getEmail(), invitation.getReceiver(), invitation.getApartmentId());
+        return new ResponseEntity<>(new MessageResponse("Invitation sent."), HttpStatus.OK);
+    }
+
+    @GetMapping("/invitation")
+    ResponseEntity<Object> getInvitations(@RequestHeader("Authorization") String token) {
+        JwtData jwtData = jwtService.readToken(token);
+        return new ResponseEntity<>(inviteService.getInvitations(jwtData.getEmail()), HttpStatus.OK);
+    }
+
+    @PostMapping("/invitation")
+    ResponseEntity<Object> resolveInvitation(@RequestBody Invitation invitation, @RequestHeader("Authorization") String token) {
```

Comment:
I wouldn't use Invitation class as Request body, cos it has a lot of redundant fields. 
I think two methods might be better (easier)
/invitation/{id}/accept
and
/invitation/{id}/reject
with no request body at all.